integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/standardpartition/StandardPartitionTableQueryTestCase.scala

```diff
@@ -189,16 +189,15 @@ class StandardPartitionTableQueryTestCase extends QueryTest with BeforeAndAfterA
   test("badrecords on partition column") {
     sql("create table badrecordsPartition(intField1 int, stringField1 string) partitioned by (intField2 int) stored by 'carbondata'")
     sql(s"load data local inpath '$resourcesPath/data_partition_badrecords.csv' into table badrecordsPartition options('bad_records_action'='force')")
-    sql("select count(*) from badrecordsPartition").show()
     checkAnswer(sql("select count(*) cnt from badrecordsPartition where intfield2 is null"), Seq(Row(9)))
     checkAnswer(sql("select count(*) cnt from badrecordsPartition where intfield2 is not null"), Seq(Row(2)))
   }
 
   test("badrecords fail on partition column") {
     sql("create table badrecordsPartitionfail(intField1 int, stringField1 string) partitioned by (intField2 int) stored by 'carbondata'")
+    //TODO: check message
```

Comment:
you can remove todo and check for the standard bad record message from exception

Reply:
this issue need to be discussed, should throw BadRecordFoundException and check message. you can check the ignore test case